queries.sql

```diff
@@ -81,3 +81,20 @@ SELECT species, AVG(escape_attempts) AS total_escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN '1990,01,01'::date AND '2000,12,31'
 GROUP By species; 
+
+
+--PROJECT 3
+-- What animals belong to Melody Pond?
+SELECT name from animals WHERE owner_id = 4
+-- List of all animals that are pokemon (their type is Pokemon).
```

Comment:
- [ ] The essence of learning this milestone is to practice the use of the `JOIN` keyword in PSQL. Currently, you are selecting the names of animals using the owner_id column. Kindly use a query that will require you to select the different relevant tables using the `JOIN` keyword. 👍 💻

#### Please, ensure that the result of the query is the correct one and attach the screenshot to your pull request. 😎